resources/views/account/accountUpdate.blade.php

```diff
@@ -0,0 +1,139 @@
+@extends('layouts.master')
+
+@section('body')
+    <div class="container">
+        @if(session('nothing'))
+            <div class="alert alert-warning" role="alert">
+                {{ session('nothing') }}
+            </div>
+        @endif
```

Comment:
Ok for now, but might want to think about putting this in `resources/views/layouts/alerts.blade.php` instead.